File: T3/codigos2122/T03_Inicio/app/src/main/java/com/example/t03_inicio/SecondActivity.java
package com.example.t03_inicio;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
        Log.v("ciclo_vida","Pasando por onCreate");
    }

    @Override
    protected void onStart() {
        super.onStart();
        Log.v("ciclo_vida","Pasando por onStart");
    }

    @Override
    protected void onRestart() {
        super.onRestart();
        Log.v("ciclo_vida","Pasando por onRestart");
    }

    @Override
    protected void onPause() {
        super.onPause();
        Log.v("ciclo_vida","Pasando por onPause");
    }

    @Override
    protected void onResume() {
        super.onResume();
        Log.v("ciclo_vida","Pasando por onResume");
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        Log.v("ciclo_vida","Pasando por onDestroy");
    }

    @Override
    protected void onStop() {
        super.onStop();
        Log.v("ciclo_vida","Pasando por onStop");
    }
}

File: Kotlin/T2/codigos_ret/T2_Estados/app/src/main/java/com/example/t2_estados/SecondActivity.kt
package com.example.t2_estados

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.ImageView
import androidx.appcompat.widget.AppCompatImageView
import androidx.appcompat.widget.AppCompatTextView

class SecondActivity : AppCompatActivity() {

    private lateinit var imagen: AppCompatImageView
    private lateinit var textoContador: AppCompatTextView
    private var contador: Int = 0;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        val bundle = intent.extras
        contador = bundle?.getInt("contador") ?: 0
        instancias();
        textoContador.text = contador.toString()
        imagen.setImageResource(R.drawable.ejemplo)
    }

    private fun instancias() {
        textoContador = findViewById(R.id.texto_contador_comunicado)
        imagen = findViewById(R.id.imagen);
    }
}